commands/http/client.go

```diff
@@ -117,6 +115,8 @@ func (c *client) Send(req cmds.Request) (cmds.Response, error) {
 		// using the overridden JSON encoding in request
 		res, err := getResponse(httpRes, req)
 		if err != nil {
+			//log.Error("ERROR ERROR ERROR")
+			httpRes.Body.Close()
```

Comment:
Actually no, nvm, in mid-flight case when the response hasn't arrived, this does not matter.

From lsof, the sockets are simply stuck in `CLOSE_WAIT` state, which means the `CancelRequest` is just not handled.